.github/workflows/release.yml

```diff
@@ -85,3 +85,7 @@ jobs:
         run: |
           cd packages/gestalt
           yarn publish --registry=https://registry.npmjs.org --no-git-tag-version --new-version ${{ steps.release.outputs.VERSION }}
+          cd packages/gestalt-datepicker
```

Comment:
I added this here for the npm release. I believe this is the automatic workflow that would publish to npm.